File: src/IsbInterop/Tasks/Proxies/JobInfo.vb
Namespace Tasks.Proxies

  ''' <summary>
  ''' Обертка над IJobInfo.
  ''' </summary>
  Friend Class JobInfo
    Inherits CustomJobInfo
    Implements IJobInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwIJobInfo">COM-объект IJobInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwIJobInfo As Object, scope As IScope)
      MyBase.New(rcwIJobInfo, scope)
    End Sub
  End Class
End Namespace

File: src/IsbInterop/Tasks/Proxies/JobInfo.cs
namespace IsbInterop.Tasks.Proxies
{
  /// <summary>
  /// Обертка над IJobInfo.
  /// </summary>
  internal class JobInfo : CustomJobInfo, IJobInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIJobInfo">COM-объект IJobInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public JobInfo(object rcwIJobInfo, IScope scope) : base(rcwIJobInfo, scope) { }
  }
}
